### PR TITLE
MSI 1.1.2 release notes 

### PR DESCRIPTION
**Pull request by lorikrell**
## Purpose of this PR

Add MSI 1.1.2 release content to the modules page and release notes. Currently, the release is planned for Magento 2.3.2. Currently using msi-1-1-2 integration branch until final.

Fixes MSI Issue: https://github.com/magento-engcom/msi/issues/2195

## Affected URLs

- https://devdocs.magento.com/guides/v2.3/inventory/release-notes.html
- https://devdocs.magento.com/extensions/inventory-management/

All PR links in html comments.

whatsnew
Added Inventory Management (MSI) [1.1.2 Release Notes](https://devdocs.magento.com/guides/v2.3/inventory/release-notes.html) and updated [module information](https://devdocs.magento.com/extensions/inventory-management/)

### CODE REVIEW

File: extensions/inventory-management/index.md [hunk 1]
```diff
@@ -4,38 +4,49 @@ title: Install Inventory Management
...
 ```json
-        magento/inventory-composer-metapackage = ^1.1.0
+        magento/inventory-composer-metapackage = 1.1.2
 ```
 
+For a list of Inventory Management metapackage versions, see the [release notes]({{site.baseurl}}/guides/v2.3/inventory/release-notes.html).
+
 The {{site.data.var.im}} installation process makes the following changes to the `<Magento_installation_directory>/composer.json` file. A `1` value indicates the corresponding module is enabled.
 
+The following list includes all modules in Inventory Management 1.1.2:
```
Comment (keharper) [inline extensions/inventory-management/index.md]: The code sample below is definitely not json. It's probably php

Comment (keharper) [reply extensions/inventory-management/index.md]: I think it'd be ideal if you listed which modules are new. Also, i'm not sure `Magento_GraphQl` is actually in IM.

(low priority comment--ignore if you want)

File: extensions/inventory-management/index.md [hunk 2]
```diff
@@ -157,7 +169,7 @@ When upgrading from previous releases of {{site.data.var.im}} to the latest vers
 For the latest, update your metapackage version:
 
 ```json
```
Comment (keharper) [inline extensions/inventory-management/index.md]:
```suggestion
```php
```

File: guides/v2.3/inventory/release-notes.md [hunk 3]
```diff
@@ -4,44 +4,62 @@ title: Release Notes
...
+-   {:.bug}Known issues
 
 ### v1.1.2
 
-{{site.data.var.im}} 1.1.2 (module version: inventory-composer-metapackage = 1.1.2)  is compatible with version 2.3.2 and 2.3.1 of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. 
+{{site.data.var.im}} 1.1.2 (module version: `inventory-composer-metapackage = 1.1.2`) is supported with version 2.3.2 and compatible with version 2.3.1, and 2.3.0 of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+- {:.new} **Added a Partial Stock Transfer API** - Current bulk trasfer endpoints move all assigned quantity from an origin to destination source. The new `/rest/V1/inventory/bulk-partial-source-transfer` endpoint allows merchants to transfer partial stock from source-to-source as a bulk operation. Enter a request to the endpoint with the `sku`, `qty`, `origin_source_code`, and `destination_source_code` to transfer a specific amount of quantity. Transfers verify the source is assigned to the sku, enough quantity exists to transfer, etc. See [Inventory Bulk Actions](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/bulk-inventory.html). <!-- https://github.com/magento-engcom/msi/pull/2117 -->
```
Comment (keharper) [inline guides/v2.3/inventory/release-notes.md]:
```suggestion
- {:.new} **Added a Bulk Partial Stock Transfer Endpoint** - Current bulk trasfer endpoints move all assigned quantity from an origin to a destination source. The new `/rest/V1/inventory/bulk-partial-source-transfer` endpoint allows merchants to transfer partial stock from source-to-source as a bulk operation. Enter a request to the endpoint with the `sku`, `qty`, `origin_source_code`, and `destination_source_code` to transfer a specific amount of quantity. Transfers verify the source is assigned to the `sku`, enough quantity exists to transfer, etc. See [Inventory Bulk Actions](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/bulk-inventory.html). <!-- https://github.com/magento-engcom/msi/pull/2117 -->
```

Comment (keharper) [reply guides/v2.3/inventory/release-notes.md]: Rewrite "specific amount of quantity"

File: guides/v2.3/inventory/release-notes.md [hunk 4]
```diff
@@ -4,44 +4,62 @@ title: Release Notes
...
+
+- {:.new} **Performance improvements for sources and SSA options** -  Sorting and selecting sources during shipment caused performance degredation for stocks with high numbers of sources. This release provides significant performance improvements to list and sort available sources when reviewing and selecting SSA options in shipments. <!-- https://github.com/magento-engcom/msi/pull/2056 https://github.com/magento-engcom/msi/pull/2090 -->
+
+- {:.new} **Added GraphQL support for {{site.data.var.im}}** - This release installs a new `magento/module-inventory-graph-ql` module. The GraphQL [Products endpoint](https://devdocs.magento.com/guides/v2.3/graphql/reference/products.html) also includes the `only_x_left_in_stock` and `stock_status` attributes for {{site.data.var.im}} support. <!-- https://github.com/magento-engcom/msi/pull/2124 -->
```
Comment (keharper) [inline guides/v2.3/inventory/release-notes.md]:
```suggestion
- {:.new} **Added GraphQL support for {{site.data.var.im}}** - This release installs a new `magento/module-inventory-graph-ql` module. The GraphQL [Products endpoint](https://devdocs.magento.com/guides/v2.3/graphql/reference/products.html) now includes the `only_x_left_in_stock` and `stock_status` attributes for {{site.data.var.im}} support. <!-- https://github.com/magento-engcom/msi/pull/2124 -->
```

File: guides/v2.3/inventory/release-notes.md [hunk 5]
```diff
@@ -4,44 +4,62 @@ title: Release Notes
...
+
+- {:.new} **Simplified UI for Assigned Sources** - The Assigned Sources table in product pages has simplified content for easier updates and increased performance when displaying many sources. All sources list by source name (hover over for `source_code`).
+
+- {:.new} **Magento B2B products support** - This release fully supports B2B products and features (including Quick Order). B2B merchants can fully use Single and Multi source features for {{site.data.var.im}}, assigning products to default and custom sources and stocks.  <!-- https://github.com/magento/partners-magento2ee/pull/18 https://github.com/magento-engcom/msi/issues/1828 -->
+
+- {:.new} **Export Aggregated Stock Service** - This release provides a new export aggregated stock service (retaining reservations in the system) to support external Sales Channels like Amazon, eBay, Google Shopping ads, etc.  <!-- https://github.com/magento-engcom/msi/pull/2067 -->
+
+- {:.fix} Added `source_code` to the response for the `/rest/V1/shipments` REST endpoint. <!-- https://github.com/magento-engcom/msi/pull/2142 -->
```
Comment (keharper) [inline guides/v2.3/inventory/release-notes.md]:
```suggestion
- {:.fix} Added `source_code` to the response for the GET `/V1/shipments` REST endpoint. <!-- https://github.com/magento-engcom/msi/pull/2142 -->
```

File: extensions/inventory-management/index.md (hunk 1 above)
Comment (lorikrell) [reply extensions/inventory-management/index.md]: Will double check the module. Thanks!

**Pull request merged**